### PR TITLE
Updates `hearbeats` Source Image References

### PR DESCRIPTION
**Pull request by JeffNeff**
Fixes the mkdocs portion of updating issue #https://github.com/knative/docs/issues/4188
Fixes the mkdocs portion of issue #https://github.com/knative/docs/issues/4178

## Proposed Changes 

- Updates the referenced image of the heartbeats source

### CODE REVIEW

**Comment by snneji** [comment]
@JeffNeff, I did a search and found the heartbeats image mentioned in the following locations as well
- https://knative.dev/docs/developer/eventing/sources/sinkbinding/getting-started/#create-a-subject
- https://knative.dev/docs/developer/eventing/sources/containersource/#create-a-containersource-object

Do these locations need to be updated as well?

File: docs/developer/eventing/sources/creating-event-sources/writing-event-source-easy-way/README.md [hunk 1]
```diff
@@ -143,7 +143,7 @@ image, and push it to a container registry that your cluster can access.
       template:
         spec:
           containers:
-            - image: docker.io/myregistry/node-heartbeat-source:v1
+            - image: gcr.io/knative-nightly/knative.dev/eventing/cmd/heartbeats:latest
```
Comment (sebgoa) [inline docs/developer/eventing/sources/creating-event-sources/writing-event-source-easy-way/README.md:146]: Please revert this, the page actually explains how to write a source from scratch. It makes you build an heartbeat source image from scratch. Hence the original image repo is valid

File: docs/developer/eventing/sources/creating-event-sources/writing-event-source-easy-way/README.md [hunk 2]
```diff
@@ -224,7 +224,7 @@ SinkBinding does not create any containers. It injects the sink information to a
         spec:
           containers:
           - name: node-heartbeats
-            image: docker.io/myregistry/node-heartbeat-source:v1
+            image: gcr.io/knative-nightly/knative.dev/eventing/cmd/heartbeats:latest
```
Comment (sebgoa) [inline docs/developer/eventing/sources/creating-event-sources/writing-event-source-easy-way/README.md:227]: same here, please revert this change, the original image path is correct.

**Review by sebgoa — CHANGES_REQUESTED** [review]
